### PR TITLE
Added backtick highlight support

### PR DESCRIPTION
**Pull request by ghyatzo**

### CODE REVIEW

File: src/passes/SyntaxHighlighter.jl [hunk 1]
```diff
@@ -123,7 +123,7 @@ function (highlighter::SyntaxHighlighterSettings)(crayons::Vector{Crayon}, token
                 crayons[i] = cscheme.keyword
             end
         # "foo"
-        elseif kind(t) == Tokens.STRING || kind(t) == Tokens.TRIPLE_STRING || kind(t) == Tokens.CHAR
+        elseif kind(t) == Tokens.STRING || kind(t) == Tokens.TRIPLE_STRING || kind(t) == Tokens.CHAR || kind(t) == Tokens.CMD
```
Comment (KristofferC) [inline src/passes/SyntaxHighlighter.jl]: `TRIPLE_CMD` as well?

Comment (ghyatzo) [reply src/passes/SyntaxHighlighter.jl]: forgot about that. Just added. Also added backtick matching similar to single and double quotes.

**Comment by ghyatzo** [comment]
might be wise to address #192, since we are at it. Is it in `BracketInserter.jl`?

**Comment by ghyatzo** [comment]
Merged with #193

**Pull request closed without merging**